### PR TITLE
Create release_notes.md

### PR DESCRIPTION
**Pull request by mcourtot**
added release notes

### CODE REVIEW

File: doc/release_notes.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+This release addresses several outstanding tickets:
+
+- A flat file (CSV) version of DUO is now being created at release time. This file is available from http://purl.obolibrary.org/obo/duo/duo.csv Despite being versioned this file does not include the version explicitely: a ticket has been created to do this in the next release, https://github.com/EBISPOT/DUO/issues/40
+- HMB has been moved to be a child of GRU and sibling of GRU-CC : HMB definition doesn't include clinical care and the hierarchy was therefore updated to reflect this.
+- The term 'consent code' has been relabeled as 'data use limitation'. To optimise search the term 'consent code' has been retained as an alternative term.
+- The term 'consent code primary category' and 'secondary category' have been deprecated: they were not used to tag datasets but instead as patterns for DUO codes capture. Those patterns are institue specific and would be better reflected by Schemablocks constraints specific to a repository.
+
+
+Minnor fixes
+- Addition of xsd:dtaatype to text field, as per https://github.com/EBISPOT/DUO/issues/33. Reported and fixed by @haoyuanli
```
Comment (rushtong) [inline doc/release_notes.md]: typo:
```suggestion
- Addition of xsd:datatype to text field, as per https://github.com/EBISPOT/DUO/issues/33. Reported and fixed by @haoyuanli
```

File: doc/release_notes.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+# DUO - Release notes
+
+## v2020-02-03
+
+This release addresses several outstanding tickets:
+
+- A flat file (CSV) version of DUO is now being created at release time. This file is available from http://purl.obolibrary.org/obo/duo/duo.csv Despite being versioned this file does not include the version explicitely: a ticket has been created to do this in the next release, https://github.com/EBISPOT/DUO/issues/40
+- HMB has been moved to be a child of GRU and sibling of GRU-CC : HMB definition doesn't include clinical care and the hierarchy was therefore updated to reflect this.
+- The term 'consent code' has been relabeled as 'data use limitation'. To optimise search the term 'consent code' has been retained as an alternative term.
+- The term 'consent code primary category' and 'secondary category' have been deprecated: they were not used to tag datasets but instead as patterns for DUO codes capture. Those patterns are institue specific and would be better reflected by Schemablocks constraints specific to a repository.
```
Comment (rushtong) [inline doc/release_notes.md]: typo:
```suggestion
- The term 'consent code primary category' and 'secondary category' have been deprecated: they were not used to tag datasets but instead as patterns for DUO codes capture. Those patterns are institute specific and would be better reflected by Schemablocks constraints specific to a repository.
```

File: doc/release_notes.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+
+# DUO - Release notes
+
+## v2020-02-03
+
+This release addresses several outstanding tickets:
+
+- A flat file (CSV) version of DUO is now being created at release time. This file is available from http://purl.obolibrary.org/obo/duo/duo.csv Despite being versioned this file does not include the version explicitely: a ticket has been created to do this in the next release, https://github.com/EBISPOT/DUO/issues/40
```
Comment (solideoglori) [inline doc/release_notes.md]:
```suggestion
- A flat file (CSV) version of DUO is now being created at release time. This file is available from http://purl.obolibrary.org/obo/duo/duo.csv Despite being versioned this file does not include the version explicitly: a ticket has been created to do this in the next release, https://github.com/EBISPOT/DUO/issues/40
```

**Comment by mcourtot** [comment]
Thanks @rushtong and @solideoglori for the feedback. @rushtong @jlawsonbroad would you mind approving the PR? These will be our first release notes as well, so wanted to be transparent and make sure the 'group' was ok with that.

**Pull request merged**